lbrynet/core/utils.py

```diff
@@ -97,11 +95,5 @@ def check_connection(server="www.lbry.io", port=80):
         return False
 
 
-def setup_certs_for_windows():
```

Comment:
pyinstaller handles bundling the cacert just fine and so we don't need to set the environment variable.